### DISCUSSION TITLE
Custom views are displayed incorrectly

### DISCUSSION THREAD

**Discussion by AndyRaphael** [imfing/hextra#307] [Q&A]
I tried to create a custom view by adding a list.html to the layouts/docs folder and copied the content from the hextra\layouts\docs\list.html file. However, this causes the page to render incorrectly. all the content is displayed on top of one another. Instead of the content being displayed from left to right, e.g. "sidebar | content | toc", it shows them top to bottom.
"Sidebar
toc
content"

**Comment by imfing** [comment]
Hi can you provide an example repository or branch to reproduce this issue?

**Comment by AndyRaphael** [comment]
Sorry, no, I do not have a public repo. However, I started a new project and this time the custom view worked as expected.
